packages/plugin-telemetry/src/polyfills.ts

```diff
@@ -0,0 +1,14 @@
+export function sendBeacon (endpoint: string, body?: string): Promise<Response> | undefined {
+  if (typeof navigator !== 'undefined') {
+    navigator.sendBeacon(endpoint, body)
+    return
+  }
+
+  fetch(endpoint, {
+    method: 'POST',
+    body,
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  })
```

Comment:
We should capture the error so the nodejs process doesn't crash.

Reply:
I'll just add a catch block here, we'll also need to discuss how to manage retries later